sdks/java/io/hbase/pom.xml

```diff
@@ -52,29 +50,6 @@
               </goals>
               <configuration>
                 <createDependencyReducedPom>false</createDependencyReducedPom>
```

Comment:
On the other hand I have to confess I am starting to feel uneasy about using ByteString in the public API of the IO (even if I understand the advante of having an immutable byte array representation).